### ISSUE TITLE
Please make a new release

### ISSUE THREAD

**Issue by mgorny** [agronholm/typeguard#257]
**Is your feature request related to a problem? Please describe.**
The 2.13.3 seems to suffer test regressions on Python 3.10.4:

```
FAILED tests/test_typeguard.py::TestTypeChecked::test_typed_dict[correct] - TypeError: TypedDict does not support instance and class...
FAILED tests/test_typeguard.py::TestTypeChecked::test_typed_dict[missing_x] - AssertionError: Regex pattern 'required key\\(s\\) \\(...
FAILED tests/test_typeguard.py::TestTypeChecked::test_typed_dict[wrong_y] - AssertionError: Regex pattern 'type of dict item "y" for...
FAILED tests/test_typeguard.py::TestTypeChecked::test_typed_dict[missing_y_error] - AssertionError: Regex pattern 'required key\\(s\...
FAILED tests/test_typeguard.py::TestTypeChecked::test_typed_dict[missing_y_ok] - TypeError: TypedDict does not support instance and ...
FAILED tests/test_typeguard.py::TestTypeChecked::test_typed_dict[wrong_x] - AssertionError: Regex pattern 'type of dict item "x" for...
FAILED tests/test_typeguard.py::TestTypeChecked::test_typed_dict[unknown_key] - AssertionError: Regex pattern 'extra key\\(s\\) \\("...
```

**Describe the solution you'd like**
The tests seem to pass cleanly on `master`, so a new release should help ;-).

**Describe alternatives you've considered**
I've tried using `git bisect` to find a commit fixing the problem but without success. I suspect the merge commits are breaking that.

**Additional context**
n/a

**Comment by agronholm** [comment]
Note that the changes on master warrant a major version bump. I will check what needs to happen for a new release, but I have quite a lot going on w/ my other projects too.

**Comment by agronholm** [comment]
By the way, the fix for the above error that you might be looking for was in #243.

**Comment by agronholm** [comment]
I've just released v3.0.0b1.
